lib/board.rb

```diff
@@ -0,0 +1,19 @@
+require '../lib/player.rb'
```

Comment:
- [ ] Remove this line or correct it because it is blocking the game to start, the path is broken resulting in an error. I tried to remove the line along with the `< Play` from the `Board` class definition and the game works perfectly.